src/Symfony/Component/HttpKernel/Attribute/MapRequestPayload.php

```diff
@@ -22,7 +23,9 @@
 class MapRequestPayload extends ValueResolver
 {
     public function __construct(
+        public readonly array|string|null $format = null,
         public readonly array $context = [],
+        public readonly string|GroupSequence|array|null|false $groups = null,
```

Comment:
why `false`?

Reply:
does it have sense to name it `validationGroups` in order to not mix with Serialization Groups?

Reply:
@nicolas-grekas fixed, thanks for triggering it. It was a wrong copy&paste. In my local solution I'm able to disable groups defined from a preset, I removed it from this PR as it has no mean for now.

@Koc changed, thanks! Changed existing `context` to `$deserializationContext` as well to keep consistency. Should we do the same in `MapQueryString`?